### ISSUE TITLE
verbs () take forever to be excuted

### ISSUE THREAD

**Issue by py4object** [nlp-compromise/compromise#373]
```
var ex2='To which platforms can chatbots be  integrated?';
var r=nlp(ex2);
console.log(r.verbs());
```
seems to be stuck in an infinite loop

**Comment by spencermountain** [comment]
oh wow,
thanks for this. looks like it's even sensitive to the question mark.
will check it out today, and fit it in the next release

**Comment by spencermountain** [comment]
fixed in 10.5.0 :rocket:
